cypress/e2e/avengersTest.cy.js

```diff
@@ -0,0 +1,69 @@
+describe('template spec', () => {
```

Comment:
Some of the review comments for the Flarum task are also related to this. Please consider making the following changes:

Assigning selectors to constants and moving them to dedicated class/classes.
Moving selectors to dedicated and specific functions/commands.
Using test data with fixtures without setting them as environment variables.
Changing assertions for the 'existing' of elements to 'visibility.